### DISCUSSION TITLE
Is vsomeip supported for Apple (MacOS) systems?

### DISCUSSION THREAD

**Discussion by srchaitu2008** [COVESA/vsomeip#687] [Q&A] [answered]
Hi ,

I am trying to compile vsomeip on Apple(MacOS) systems and observing lot of compilation errors. Is vsomeip supported for Apple (MacOS) systems?

Thanks,
Ravi

**Comment by srchaitu2008** [comment]
Hi @fcmonteiro @goncaloalmeida @reymor ,

Can I get help with this please?

Thanks,
Ravi.

**Reply by fcmonteiro** [reply, marked as answer]
Hello, currently we do not support MacOS.
However, I believe the code adaptation shouldn't be that difficult.
You can try adding `__APPLE__` to the ifdef/ifndef we have in the code. eg;
`#if defined(__linux__) || defined(ANDROID) || defined(__QNX__)`
